src/lib/Libifl/pbsD_connect.c

```diff
@@ -171,6 +171,45 @@ get_hostsockaddr(char *host, struct sockaddr_in *sap)
 	return -1;
 }
 
+/**	
+ * @brief	
+ *	-hostnmcmp - compare two hostnames, allowing a short name to match a longer	
+ *	version of the same	
+ *	
+ * @param[in] s1 - hostname1	
+ * @param[in] s2 - hostname2	
+ *	
+ * @return	int	
+ * @retval	1	success	
+ * @retval	0	failure	
+ *	
+ */	
+static int	
+hostnmcmp(char *s1, char *s2)	
```

Comment:
Can you make use of is_same_host() instead?

Reply:
There was a special code to support "localhost" & "localhost.domain" in hostnmcmp(). we cant use is_same_host() in this place.

Reply:
Does that already guaranteed by get_fullhostname() within is_same_host?
If not why don't you extend is_same_host instead of adding a new function?